ci/deploy.yaml

```diff
@@ -0,0 +1,54 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: theorf/google-cloud-sdk-helm
+inputs:
+  - name: eq-questionnaire-runner-repo
+params:
+  GOOGLE_APPLICATION_CREDENTIALS: /root/gcloud-service-key.json
+  SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+  PROJECT_NAME:
+  EQ_KEYS_FILE:
+  EQ_SECRETS_FILE:
+  DOCKER_REGISTRY:
+  IMAGE_TAG:
+  GOOGLE_TAG_MANAGER_ID:
+  GOOGLE_TAG_MANAGER_AUTH:
+  GOOGLE_TAG_MANAGER_PREVIEW:
+  COOKIE_SETTINGS_URL:
+  REQUESTED_CPU_PER_POD:
+  ROLLING_UPDATE_MAX_UNAVAILABLE:
+  ROLLING_UPDATE_MAX_SURGE:
+  MIN_REPLICAS:
+  MAX_REPLICAS:
+  TARGET_CPU_UTILIZATION_PERCENTAGE:
+  REGION:
+run:
+  path: bash
+  args:
+    - -exc
+    - |
+      apt-get install -y procps
+
+      cat >$GOOGLE_APPLICATION_CREDENTIALS <<EOL
+      $SERVICE_ACCOUNT_JSON
+      EOL
+
+      gcloud auth activate-service-account --key-file ${GOOGLE_APPLICATION_CREDENTIALS}
+
+      cd eq-questionnaire-runner
+
+      helm init --client-only
+      helm plugin install https://github.com/rimusz/helm-tiller
+
+      PROJECT_ID=($(gcloud projects list --filter=name="${PROJECT_NAME}" --format="value(PROJECT_ID)"))
+
+      [[ "${#PROJECT_ID[@]}" = 1 ]] || exit 1
+
+      SUBMISSION_BUCKET_NAME=${PROJECT_ID}-survey-runner-submission
+
+      gcloud container clusters get-credentials survey-runner --region ${REGION} --project ${PROJECT_ID}
+      ./k8s/deploy_credentials.sh
```

Comment:
If we want to use this task for all pipelines we'll need to move the credentials creation into a separate task as this script is deploying the test keys and secrets.

Reply:
This doesn't need to be a separate task as it is configurable?.
We can specify `EQ_KEYS_FILE` and `EQ_SECRETS_FILE` therefore this can just be passed through concourse making it useable for both formal and dev environments.

Reply:
The keys and secrets used by the script have to exist in the runner repo, which won't be possible for anything other than the test keys.